static/app/components/devtoolbar/components/panelRouter.tsx

```diff
@@ -44,6 +45,8 @@ export default function PanelRouter() {
           <PanelReleases />
         </AnalyticsProvider>
       );
+    case 'replay':
+      return <PanelReplay />;
```

Comment:
let's add an `<AnalyticsProvider>` wrapper around here so we can track when users press record!